src/cmds/qmgr.c

```diff
@@ -1204,6 +1204,17 @@ make_connection(char *name)
 	else
 		PSTDERR1("qmgr: cannot connect to server %s\n", name)
 
+	if (msvr_mode()) {
+		int i;
+		svr_conn_t *svr_connections = get_conn_servers(connection);
```

Comment:
feels like a null terminated array is better than having to asking for the number of connections.

Reply:
Yes, we can do that as well. There are places where we need to make use of num_of_servers even if we have a null-terminated array. We make use of num_of_servers in case of finding a random server index. Also when finding the server to connect to based on the object id (job.servername). Using no_of_servers does cater all our use cases.
Also no of servers is a global var accessed only using a getter. Let me know if you still thinks it needs to be changed.

Reply:
its fine for now